tests/server/test_calling_routes.py

```diff
@@ -485,6 +485,25 @@ def test_update_recording_name__returns_ok_if_recording_name_doesnt_exist(
     assert response.status_code == 200
 
 
+@pytest.mark.parametrize(
+    "test_new_file_name",
+    ["NormalName", "  TestName", "TestName  ", "  TestName  "],
+)
+def test_update_recording_name__trims_spaces_from_start_and_end_beta_2_mode(
+    test_new_file_name, client_and_server_manager_and_shared_values, mocker
+):
+    test_client, _, shared_values_dict = client_and_server_manager_and_shared_values
+    shared_values_dict["beta_2_mode"] = True
+    shared_values_dict["config_settings"]["recording_directory"] = "/test/recording/directory"
+
+    mocker.patch.object(os.path, "exists", return_value=False)
+
+    response = test_client.post(
+        f"/update_recording_name?new_name={test_new_file_name}&default_name=old_name&snapshot_enabled=true"
+    )
+    assert response.status_code == 200
```

Comment:
The purpose of this test is to make sure that the new file name gets trimmed, but the only assertion made is this one. We need to make sure that the message this route sends to `main` contains the trimmed file name. You should be able to find some other tests that do this in `tests/server/test_routes_populate_queues`, I also think it makes sense to move this test into that file